### DISCUSSION TITLE
Potential issue with multiselect from another collection

### DISCUSSION THREAD

**Discussion by ryanlanciaux** [payloadcms/payload#703] [Q&A]
I'm currently encountering an issue with a field that is a multiselect of multiple items from another collection. When editing the item, the screen flickers a lot and ONLY 10 of the selected items are ever present, even when adding an 11th. Looking at the network tab, it appears that it is making paginated calls to this record but only 10 items are ever showing up.

With the 10 items, the dropdown selector shows "None" as remaining items. However, when typing it does render additional items but with multiples of same item (4 or more even though there is only one matching record). The rendering of the list along with the items already selected continues to flicker as well.

The very strange thing about this, is only one item that references this data structure is displaying this behavior right now. I don't know if it's simply because I have exactly 10 items initially selected which is throwing the pagination for the multiselect control off, or if it's something else.

Have you ever seen this behavior and do you have any idea on how I may resolve? I'd be happy to create an issue and potentially dig through this a bit, but wanted to start here and see if there were any ideas. Thanks I really appreciate any advice you may have!

**Comment by DanRibbens** [comment]
I am currently fixing another issue with relationship fields which has to do with access control. See PR for details: https://github.com/payloadcms/payload/pull/644

After this is released I'd like you to try again and if its still an issue post the configuration of your relationship field so we can test it.
I'll @ you when its released, hoping to get it done today still.
